src/index.js

```diff
@@ -138,81 +156,62 @@ export default options => {
       wild.match(room, key =>
         send(rooms[key], clients, data, options)
     }
-  }
-
-  /**
-   * Create sender.
-   * Helper for broadcast
-   * @param {String[]} ids - spark IDs to send to
-   * @param {Object} clients - connected clients
-   * @param {*} data - the data to send
-   * @param {Object} options - same options as broadcast
-   * @api private
-   */
-
-  function send(ids, clients, data, options) {
-    const { except = [], method = 'writer', transformer } = options
-    let sent = {}
-    for (let id in ids) {
-      const socket = clients[id]
-      if (sent[id] || ~except.indexOf(id) || !socket) continue
-      transform(socket, data, method, transformer)
-      sent[id] = true
+    /**
+     * Create sender.
+     * Helper for broadcast
+     * @param {String[]} ids - spark IDs to send to
+     * @param {Object} clients - connected clients
+     * @param {*} data - the data to send
+     * @param {Object} options - same options as broadcast
+     * @api private
+     */
+
+    function send(ids, clients, data, options) {
+        const { except = [], method = 'writer', transformer } = options
```

Comment:
Agree, it did look weird when I saw it. I wasn't sure if it was a new method added in a newer version or a typo. I didnt get the chance to investigate it, it was on this branch when i started working on it